process/wg-lifecycle-documents/BEST_practices_wg_graduation_stage.md

```diff
@@ -0,0 +1,41 @@
+## Working Group graduation application
+BEST Practices Working Group
+
+### WG has met all Incubating requirements
+  * n/a
+
+### List of regular contributors
+The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
+- [Christopher "CRob" Robinson*, Intel](https://github.com/SecurityCRob)
+- [David A Wheeler, LF/OSSF](https://github.com/david-a-wheeler)
+- [Dave Russo*, Red Hat](https://github.com/drusso-rh)
+- [Randall T. Vasquez*, The Linux Foundation](https://github.com/ran-dall)
+- [Arnaud J Le Hors, IBM](https://github.com/lehors)
+- [Christine Abernathy*, F5](https://github.com/caabernathy)
+- [Daniel Applequist*, Snyk](https://github.com/Torgo)
+- [Georg Kunz, Ericsson](https://github.com/gkunz)
+- [Sarah Evans, Dell](https://github.com/sevansdell) - Security Toolbelt SIG Co-Lead
+- [John Kjell, TestifySec](https://github.com/jkjell) - Security toolbelt SIG Co-lead
+- [Thomas Nyman, Ericsson](https://github.com/thomasnyman) - C/C++ Compiler Guide SIG Lead
+- [Nell Shamrell-Harrington](https://github.com/nellshamrell) - Memory Safety SIG Lead
+- [Avishay Balter, Microsoft](https://github.com/balteravishay) - Memory Safety SIG Co-Lead
+
+
+### Governance
```

Comment:
Can you please add a link in here that points to the project governance documents (per the https://github.com/ossf/tac/pull/296 TIs Gives+Gets and updated WG template)?

Reply:
Added link to WG charter - https://github.com/ossf/wg-best-practices-os-developers/blob/main/CHARTER.md